README.md

```diff
@@ -1,52 +1,22 @@
 # zkFold Prover
 Optimized ZK provers
 
+### Setting up the environment
```

Comment:
Please specify all the prerequisites for running `run.sh`. I needed to install `cbindgen` with `cargo install --force cbindgen`, for example.

Reply:
I have added main points for setting up the environment